ConsoleApp1/ConsoleApp1/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Вариант 3");
+            var x = SquareSum(2, SquareSum(3, SquareSum(4, SquareSum(5, 0))));
+            Console.WriteLine($"x = {Math.Round(x, 3)}");
```

Comment:
Проще использовать форматирование
```suggestion
            Console.WriteLine($"x = {x:F3}");
```